gspread/worksheet.py

```diff
@@ -3336,3 +3338,48 @@ def add_validation(
         }
 
         return self.client.batch_update(self.spreadsheet_id, body)
+
+    def expand_table(
```

Comment:
this function could have a simplified name. I am not sure "expand_table" is that user-friendly? perhaps just "expand"? or maybe it is good

Reply:
I checked and in the source library where this idea comes from it's called `expand()` We could use the same name so it matches.